blockproducer/metastate_test.go

```diff
@@ -1037,10 +1037,11 @@ func TestMetaState(t *testing.T) {
 				err = up.Sign(privKey1)
 				So(err, ShouldBeNil)
 				err = ms.apply(&up)
-				So(errors.Cause(err), ShouldEqual, ErrInvalidPermission)
+				So(err, ShouldBeNil)
 				// test permission update
 				// addr1(admin) update addr3 as admin
 				up.TargetUser = addr3
+				up.Nonce += 1
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
should replace up.Nonce += 1 with up.Nonce++